ZKSyncSDK/Classes/Signer/Ethereum/EthSigner.swift

```diff
@@ -14,12 +14,16 @@ public protocol EthSigner {
     
```

Comment:
As this post mentions "open access applies only to classes and class members". EthSigner is a protocol so it can be implemented outside library with public access. But on the other hand EthSignature that it has to return in number of methods did not have public initialiser which makes it impossible to construct it. So added public initialiser to it